src/App.tsx

```diff
@@ -1,35 +1,61 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
```

Comment:
Please update src/App.css to make the to do list look better.

Files:
- src/App.css